### PR TITLE
fix docs for 2.067

### PR DESCRIPTION
**Pull request by 9il**
@MartinNowak  please add this doc fix to the release.
`sumOfLog2s` is new function.

### CODE REVIEW

**Comment by quickfur** [comment]
Looks good. But why are the comments in `//` not included in the docs?

**Comment by 9il** [comment]
I think this information is redundant for user.

**Comment by quickfur** [comment]
Really? I think it's relevant, it explains why they should use this function instead of adding the logs themselves.

**Comment by 9il** [comment]
fixed

File: std/numeric.d [hunk 1]
```diff
@@ -1647,7 +1647,9 @@ unittest
 }
 
 /**
-Computes accurate sum of binary logarithms of input range $(D r).
+Computes sum of binary logarithms of input range $(D r).
+A worst-case error of roughly O(εN), average O(εN^^0.5).
+An absolute error of roughly is much less then naive sum of log2s has.
  */
```
Comment (MartinNowak) [inline std/numeric.d]: Compute the sum of binary logarithms of the input range r.
The error of this method is much smaller than with a naive sum of log2.
It is O(EN^^0.5) on average and O(EN) in the worst-case as opposed to O(...) for the naive sum.

Comment (9il) [inline std/numeric.d]: Thanks!
Naive sum is O(EN^^0.5) on average and O(EN) in the worst-case. But absolute errors are different. I think I can remove all information about O(...).

**Comment by 9il** [comment]
rebased

**Pull request merged**